_layouts/base.html

```diff
@@ -9,13 +9,13 @@
   {% endif %}
 {% endif %}
 {% if page.title %}
-  {% assign page.title page_title_starts_with_quarkus = page.title | startswith: 'Quarkus -' %}
+  {% assign page.title page_title_ends_with_quarkus = page.title | endswith: '- Quarkus' %}
 {% else %}
-  {% assign page.title page_title_starts_with_quarkus = false %}
+  {% assign page.title page_title_ends_with_quarkus = false %}
 {% endif %}
 
 <head>
-  <title>{% unless page_title_starts_with_quarkus %}Quarkus - {% endunless %}{{ page.title }}{{ page_title_version_suffix }}</title>
+  <title>{% unless page_title_ends_with_quarkus %}{% endunless %}{{ page.title }}{{ page_title_version_suffix }} - Quarkus</title>
```

Comment:
This is incorrect. The `unless` condition should surround ` - Quarkus`. I will fix it too.

Reply:
Hmmmm, and `endswith` was not implemented so my guess is that it wasn't working at all.